Engine/AlgorithmManager.cs

```diff
@@ -1187,6 +1189,13 @@ private static void ProcessSplitSymbols(IAlgorithm algorithm, List<Split> splitW
                 var configs = algorithm.SubscriptionManager.SubscriptionDataConfigService
                     .GetSubscriptionDataConfigs(security.Symbol);
 
+                if (configs.Count == 0)
+                {
+                    Log.Trace($"AlgorithmManager.ProcessSplitSymbols(): {_algorithm.Time} - No subscriptions found for {security.Symbol}" +
```

Comment:
@jaredbroad  I only left this `Log.Trace` just in case there is another bug similar to the one fixed in this PR. If the `if` statement evaluates to `true`, the algorithm will terminate anyway with a `Sequence contains no elements` exception, thrown on the next line.

Maybe better change it to a `throw` ?